pkg/agent/proxy/metrics/metrics.go

```diff
@@ -0,0 +1,117 @@
+			ConstLabels: map[string]string{"ipFamily": "v4"},
+			Name:        "sync_proxy_rules_duration_seconds",
+			Help:        "SyncProxyRules latency in seconds of Antrea Proxy",
+			Buckets:     kmetrics.ExponentialBuckets(0.001, 2, 15),
+		},
+	)
+	ServicesInstalledTotal = kmetrics.NewGauge(
+		&kmetrics.GaugeOpts{
+			Subsystem:   subsystemProxy,
+			ConstLabels: map[string]string{"ipFamily": "v4"},
+			Name:        "total_services_installed",
+			Help:        "The number of Services installed at Antrea Proxy",
+		},
+	)
+	EndpointsInstalledTotal = kmetrics.NewGauge(
+		&kmetrics.GaugeOpts{
+			Subsystem:   subsystemProxy,
+			ConstLabels: map[string]string{"ipFamily": "v4"},
+			Name:        "total_endpoints_installed",
+			Help:        "The number of Endpoints installed at Antrea Proxy",
+		},
+	)
+	ServicesUpdatesTotal = kmetrics.NewCounter(
+		&kmetrics.CounterOpts{
+			Subsystem:   subsystemProxy,
+			ConstLabels: map[string]string{"ipFamily": "v4"},
+			Name:        "total_services_update",
+			Help:        "The cumulative number of updates invocation to OnServiceUpdate at Antrea Proxy",
+		},
+	)
+	EndpointsUpdatesTotal = kmetrics.NewCounter(
+		&kmetrics.CounterOpts{
+			Subsystem:   subsystemProxy,
+			ConstLabels: map[string]string{"ipFamily": "v4"},
+			Name:        "total_endpoints_update",
+			Help:        "The cumulative number of updates invocation to OnEndpointsUpdate at Antrea Proxy",
+		},
+	)
+
+	SyncProxyLatencyV6 = kmetrics.NewHistogram(
+		&kmetrics.HistogramOpts{
+			Subsystem:   subsystemProxy,
+			ConstLabels: map[string]string{"ipFamily": "v6"},
+			Name:        "sync_proxy_rules_duration_seconds",
+			Help:        "SyncProxyRules latency in seconds of Antrea Proxy",
+			Buckets:     kmetrics.ExponentialBuckets(0.001, 2, 15),
+		},
+	)
+	ServicesInstalledTotalV6 = kmetrics.NewGauge(
+		&kmetrics.GaugeOpts{
+			Subsystem:   subsystemProxy,
+			ConstLabels: map[string]string{"ipFamily": "v6"},
+			Name:        "total_services_installed",
+			Help:        "The number of Services installed at Antrea Proxy",
+		},
+	)
+	EndpointsInstalledTotalV6 = kmetrics.NewGauge(
+		&kmetrics.GaugeOpts{
+			Subsystem:   subsystemProxy,
+			ConstLabels: map[string]string{"ipFamily": "v6"},
+			Name:        "total_endpoints_installed",
+			Help:        "The number of Endpoints installed at Antrea Proxy",
+		},
+	)
+	ServicesUpdatesTotalV6 = kmetrics.NewCounter(
+		&kmetrics.CounterOpts{
+			Subsystem:   subsystemProxy,
+			ConstLabels: map[string]string{"ipFamily": "v6"},
+			Name:        "total_services_update",
+			Help:        "The cumulative number of updates invocation to OnServiceUpdate at Antrea Proxy",
+		},
+	)
+	EndpointsUpdatesTotalV6 = kmetrics.NewCounter(
+		&kmetrics.CounterOpts{
+			Subsystem:   subsystemProxy,
+			ConstLabels: map[string]string{"ipFamily": "v6"},
+			Name:        "total_endpoints_update",
+			Help:        "The cumulative number of updates invocation to OnEndpointsUpdate at Antrea Proxy",
```

Comment:
Feel the method name in the Help field makes it too specific and could be confusing. 
Why not simply "The cumulative number of endpoint updates at Antrea Proxy"?

Reply:
I used to think the `endpoint updates` may be easily understood as updates on Kubernetes Endpoints resources. But a second thought, you're right. Updated.